include/SugarCache/SugarCache.php

```diff
@@ -143,8 +143,10 @@ public static function cleanOpcodes($full_reset = false)
         // Zend OPcache
         if (
             extension_loaded('Zend OPcache') &&
+            ((function_exists('opcache_get_status') &&
             ($opcache_status = opcache_get_status(false)) !== false &&
-            $opcache_status['opcache_enabled'] && $full_reset
+            $opcache_status['opcache_enabled'] ) || ini_get('opcache.enable') ) &&
+            $full_reset
```

Comment:
Hi @JanSiero,

Thank you for your PR.

This condition is getting hard to read, could we do something like the following, please?

```php
$opcacheLoaded = extension_loaded('Zend OPcache');
$opcache_status_enabled = false;

if(function_exists('opcache_get_status')) {
    $opcache_status = opcache_get_status(false);
    $opcache_status_enabled = $opcache_status !== false && ($opcache_status['opcache_enabled'] ?? false);
}

$ini_opcache_enabled = ini_get('opcache.enable');

if (
    opcacheLoaded &&
    (opcache_status_enabled || $ini_opcache_enabled) &&
    $full_reset
) {
    if (!opcache_reset()) {
        LoggerManager::getLogger()->error("OPCache - could not reset");
    }
}
```

Reply:
Hi @clemente-raposo your code always retrieves the opcache status by ini_get, even when opcache_get_status was successful, although it was meant as a fallback.

To reduce the nr. of variables, and improve readability I'd propose this code. Can you assess this? Would you like me to adapt my pull request accordingly?

```php
        // Zend OPcache
        if (extension_loaded('Zend OPcache') {
            $opcache_status_enabled = false;
            
            if(function_exists('opcache_get_status')) {
                $opcache_status = opcache_get_status(false);
                $opcache_status_enabled = $opcache_status !== false && ($opcache_status['opcache_enabled'] ?? false);
            } else {
                $opcache_status_enabled = ini_get('opcache.enable');
            }
            
            if ($opcache_status_enabled && $full_reset) {
                if (!opcache_reset()) {
                    LoggerManager::getLogger()->error("OPCache - could not reset");
                }
            }
        }
```

Reply:
Hi @clemente-raposo I revised my code with a new push, I hope this helps.

Reply:
Hi @JanSiero thank you!